packages/sdk-ui-ts/src/utils/bridge.ts

```diff
@@ -432,6 +439,12 @@ export const cosmosNativeDenomsFromChainId = {
     tokenType: TokenType.Ibc,
     denom: 'inj',
   },
+
+  [CosmosChainId.Noble]: {
+    ...tokenMetaUtils.getMetaBySymbol('USDCnb'),
+    tokenType: TokenType.Ibc,
+    denom: 'USDTnb',
```

Comment:
denom should be `ibc/2CBC2EA121AE42563B08028466F37B600F2D7D4282342DE938283CC3FB2BC00E`